src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/BuildPropertyPage/DefaultPlatformPropertyValueProvider.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties
+{
+    [ExportInterceptingPropertyValueProvider("PlatformTarget", ExportInterceptingPropertyValueProviderFile.ProjectFile)]
+    internal sealed class DefaultPlatformPropertyValueProvider : InterceptingPropertyValueProviderBase
+    {
+        private readonly ConfiguredProject _configuredProject;
+
+        [ImportingConstructor]
+        public DefaultPlatformPropertyValueProvider(ConfiguredProject configuredProject)
+        {
+            _configuredProject = configuredProject;
+        }
+
+        public override Task<string> OnGetEvaluatedPropertyValueAsync(string propertyName, string evaluatedPropertyValue, IProjectProperties defaultProperties)
+        {
+            return Task.FromResult(GetPlatformValue(evaluatedPropertyValue));
+        }
+
+        public override Task<string> OnGetUnevaluatedPropertyValueAsync(string propertyName, string unevaluatedPropertyValue, IProjectProperties defaultProperties)
+        {
+            return Task.FromResult(GetPlatformValue(unevaluatedPropertyValue));
+        }
+
+        private string GetPlatformValue(string unevaluatedPropertyValue)
```

Comment:
This argument may be either evaluated or unevaluated. I suggesting changing the parameter name to `value` to avoid confusion.

Reply:
done.